source/common/common/utility.cc

```diff
@@ -67,6 +69,55 @@ bool StringUtil::atoul(const char* str, uint64_t& out, int base) {
   }
 }
 
+absl::string_view StringUtil::ltrim(absl::string_view source) {
+  source.remove_prefix(std::min(source.find_first_not_of(" \t\f\v\n\r"), source.size()));
+  return source;
+}
+
+absl::string_view StringUtil::rtrim(absl::string_view source) {
+  source.remove_suffix(source.size() - source.find_last_not_of(" \t\f\v\n\r") - 1);
+  return source;
+}
+
+absl::string_view StringUtil::trim(absl::string_view source) { return ltrim(rtrim(source)); }
+
+bool StringUtil::find(absl::string_view source, absl::string_view delimiters,
+                      absl::string_view key_token, bool trim_whitespace) {
+  const std::vector<absl::string_view> tokens = splitToken(source, delimiters, !trim_whitespace);
```

Comment:
Why !trim_whitespace?  I could understand 'false' there give the comment you made in the other CL about lazily stripping whitespace, but if the caller of this passes in false, you'll then have splitToken strip whitespace, so that looks wrong.

And I think now that we are working with string_view, whitespace stripping is pretty fast; it's probably not worth  doing it lazily.

Reply:
I thought that would reduce computations when trimming white spaces, but that's actually wrong and you are right, string_view is pretty fast. I will change that.